### ISSUE TITLE
Unthemed powerline when switching to  *Messages* buffer while using powerline-center-evil-theme

### ISSUE THREAD

**Issue by rubin55** [milkypostman/powerline#170]
When using powerline-center-evil theme, and switching to the *Messages* buffer, the powerline theme is unapplied. Switching back to *Buffer List* or *scratch* re-enables the theme. See:

<img width="1280" alt="Screenshot 2019-03-14 at 21 19 14" src="https://user-images.githubusercontent.com/6214491/54401739-4e5d6a00-469f-11e9-80ce-8b7946fd186e.png">

And with multiple windows:

<img width="1280" alt="Screenshot 2019-03-14 at 21 17 22" src="https://user-images.githubusercontent.com/6214491/54401750-5c12ef80-469f-11e9-982b-7eea24c76a1d.png">

**Comment by milkypostman** [comment]
any thoughts on what's happening here? i don't have any idea.v

**Comment by c-alpha** [comment]
One thing that caught my eye is that all buffer names in the screen shots are surrounded by `*`. My messages buffer for instance is called `Messages`, not `*Messages*`.

Could it be the case that some other package is getting in the way here?

@rubin55, where do the `*` surrounding the buffer names come from?

**Comment by milkypostman** [comment]
* is default in emacs. i think you probably have something else changing
the name.

**Comment by c-alpha** [comment]
> \* is default in emacs. i think you probably have something else changing the name.

You are right, of course. I realised as I had submitted my comment, and have edited it immediately. Ashes to my head. :-(

The screenshots hint to ParEdit and UndoTree being used. If and when "unfavourable package combination" were a suspicion to be investigated, @rubin55 would need to set his `.emacs(.d)` and `custom.el` aside, start with nothing but powerline loaded, and keep trying other packages in turn in combination with powerline (i.e. powerline+A, powerline+B, etc.). This admittedly can be a lengthy process, but without an MWE it will be very difficult.

**Comment by rubin55** [comment]
I've moved my .emacs and .emacs.d aside and came up with a minimal-ish .emacs, see here:
[dotemacs-repoducer.txt](https://github.com/milkypostman/powerline/files/3042451/dotemacs-repoducer.txt)

To try, install that as .emacs, then `M-x package-install evil` and `M-X package-install powerline`, restart, switch to Messages buffer and observe the same bug. I'm running Emacs 26.1 on Mac OS from https://emacsformacosx.com/

**Comment by milkypostman** [comment]
is this still happening? almost two years later?
